### PR TITLE
Fil/header

### PR DESCRIPTION
**Pull request by filippociandy10**
Completed header component.

### CODE REVIEW

**Comment by dimassudjito** [comment]
Did you create the branch fresh from the develop branch or other branch or reuse old branch? I'm seeing our old commits which makes code review hard
image

File: frontend/src/components/shared/Header.js [hunk 1]
```diff
@@ -1,8 +1,43 @@
...
+                    <Typography fontSize={20} color='primary.light' mt={0.5} ml={0.5}>blueprint</Typography>
+
+                    <Box sx={{
+                        display: "flex",
+                        flexDirection: "row",
+                        flexGrow: 1,
+                        flexWrap: 'wrap',
+                        justifyContent: "flex-end",
+                    }}>
+                        <Typography mx={2} my={1}>
```
Comment (dimassudjito) [inline frontend/src/components/shared/Header.js:25]: make all of the nav text bold

File: frontend/src/components/shared/Header.js [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
 function Header() {
-    return <div>-header component placeholder-</div>
+    return (
+        <main>
+            <div>
+                <Box sx={{ display: "flex", flexDirection: "row", bgcolor: "primary.dark", p: 3 }}>
+                    <Box component="img"
+                        sx={{ height: 50, width: 45 }}
+                        src="/assets/bp-logo.png" />
+                    <Typography fontSize={20} color='primary.light' mt={0.5} ml={0.5}>blueprint</Typography>
```
Comment (dimassudjito) [inline frontend/src/components/shared/Header.js]: replace with this logo: https://drive.google.com/file/d/1fHsudAy8SiM1J1dfUp73e07BxWfhUQ12/view?usp=sharing

Comment (dimassudjito) [reply frontend/src/components/shared/Header.js]: add left margin

File: frontend/src/components/shared/Header.js (hunk 1 above)
Comment (dimassudjito) [reply frontend/src/components/shared/Header.js:25]: add right margin

File: frontend/src/components/shared/Header.js (hunk 2 above)
Comment (dimassudjito) [reply frontend/src/components/shared/Header.js]: make it link to home page

**Review by dimassudjito — CHANGES_REQUESTED** [review]
Minor formatting changes

**Comment by dimassudjito** [comment]
> Did you create the branch fresh from the develop branch or other branch or reuse old branch? I'm seeing our old commits which makes code review hard image

The footer branch doesn't have this problem, weird

File: frontend/src/components/shared/Header.js (hunk 2 above)
Comment (filippociandy10) [reply frontend/src/components/shared/Header.js]: the logo in the link doesnt have blue colors on the logo. 
image

Comment (dimassudjito) [reply frontend/src/components/shared/Header.js]: Yeah that's okay, I'll update the figma later

**Comment by dimassudjito** [comment]
@filippociandy10 I'm ready to approve, please solve the merge conflict thank you

**Review by dimassudjito — APPROVED** [review]

**Pull request merged**